src/components/IndiPick/IndiPick.tsx

```diff
@@ -0,0 +1,165 @@
+  IndiPickContainer,
+  IndiPickHeader,
+  IndiPickContents,
+  IndiPickVS,
+  //
+  IndiPickItemWrapper,
+  IndiPickLikeBox,
+  IndiPickLikeBoxContainer,
+  IndiPickLikeBoxBottom,
+  IndiPickItemBox,
+  //
+  IndiPickItemBoxImg,
+  IndiPickItemBoxContents,
+  IndiPickItemBoxName,
+  IndiPickItemBoxGenres,
+  IndiPickItemGenre,
+} from "./IndiPick.style";
+
+interface IndiPickItemProps {
+  name: string;
+  id: number;
+  like: number;
+  header_image: string;
+  genres: [string];
+  isLeft: boolean;
+  btnState: string;
+  onClick: (id: number) => void;
+}
+interface IndiPickProps {
+  indiPickList: [
+    {
+      name: string;
+      id: number;
+      like: number;
+      header_image: string;
+      genres: [string];
+    }
+  ];
+}
+
+const IndiPckItem = ({
+  name,
+  id,
+  like,
+  header_image,
+  genres,
+  isLeft,
+  btnState,
+  onClick,
+}: IndiPickItemProps) => {
+  const handleClick = () => onClick(id);
+  const [num, setNum] = useState<number>(like);
+
+  useEffect(() => {
+    setNum(like);
+  }, [like]);
+  return (
+    <IndiPickItemWrapper isLeft={isLeft}>
+      <IndiPickLikeBox btnState={btnState} onClick={handleClick}>
+        {btnState === "default" ? (
+          <IndiPickLikeBoxContainer>
+            <h4>Click!</h4>
+            <IndiPickLikeBoxBottom>
+              <Image src={LikeIcon} height={16} width={16} />
+              <p>{num}</p>
+            </IndiPickLikeBoxBottom>
+          </IndiPickLikeBoxContainer>
+        ) : btnState === "active" ? (
+          <IndiPickLikeBoxContainer>
+            <h2>Like</h2>
+            <IndiPickLikeBoxBottom>
+              <Image src={LikeIcon} height={16} width={16} />
+              <p>{num}</p>
+            </IndiPickLikeBoxBottom>
+          </IndiPickLikeBoxContainer>
+        ) : (
+          <IndiPickLikeBoxContainer>
+            <p>
+              Vote <br /> Complete
+            </p>
+          </IndiPickLikeBoxContainer>
+        )}
+      </IndiPickLikeBox>
+
+      <Link href={`/about/${id}`}>
+        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+        <a>
+          <IndiPickItemBox isLeft={isLeft}>
+            <IndiPickItemBoxImg img={header_image} />
+            <IndiPickItemBoxContents>
+              <IndiPickItemBoxName>{name}</IndiPickItemBoxName>
+              <IndiPickItemBoxGenres>
+                {genres.map((genre: string) => (
+                  <IndiPickItemGenre key={genre}>{genre}</IndiPickItemGenre>
+                ))}
+              </IndiPickItemBoxGenres>
+            </IndiPickItemBoxContents>
+          </IndiPickItemBox>
+        </a>
+      </Link>
+    </IndiPickItemWrapper>
+  );
+};
+const IndiPick = ({ indiPickList }: IndiPickProps) => {
+  const items = [...indiPickList];
+
+  const { t } = useTranslation("main");
+  const [pickState, setPickState] = useState([
+    { id: items[0].id, state: "default", like: items[0].like },
+    { id: items[1].id, state: "default", like: items[1].like },
+  ]);
```

Comment:
언제나 2개의 아이템만을 사용할테니 배열의 상태보다 상태 2개를 사용하는건 어떨까요? 아래의 onClick함수가 더 간단해 질 수 있을 것 같습니다.

Reply:
이 부분은 pickState가 true/false 처럼 2가지 상태면 괜찮을텐데, `default` `active` inactive` 3가지 상태라서,,, 오히려 setState함수가 늘어면서거 onClick 메소드가 길어지는 것 같아요.. 

예시 코드는 아래와 같은데 어떠신가요???

```suggestion
  const [pickStateLeft, setPickStateLeft] = useState({
    id: items[0].id,
    state: "default",
    like: items[0].like,
  });
  const [pickStateRight, setPickStateRight] = useState({
    id: items[1].id,
    state: "default",
    like: items[1].like,
  });

  const onClick = (id: number) => {
   // 왼쪽 선택시,
    if (pickStateLeft.id === id) {
      setPickStateLeft({
        ...pickStateLeft,
        state: "active",
        like: pickStateLeft.like + 1,
      });
      setPickStateRight({
        ...pickStateRight,
        state: "inactive",
      });
    } 
    // 오른쪽 선택시,
    else {
      setPickStateRight({
        ...pickStateLeft,
        state: "active",
        like: pickStateLeft.like + 1,
      });
      setPickStateLeft({
        ...pickStateRight,
        state: "inactive",
      });
    }
  };

```

Reply:
**전체보기** 기능도 있어서 해당 방식으로 유지해서 머지할게요..!